# <center>2213645 刘嘉硕 回归分析第五次上机作业</center>

## (1) 将数据表成方差分析模型的形式

$$
\begin{cases}
y_{ijk} = \mu + \alpha_i + \beta_j + e_{ijk}\\
e_{ijk} \sim N(0,\sigma^2)
\end{cases}\hspace{2em}
i = 1,2,3,\hspace{1em}
j = 1,2,3,4,\hspace{1em}
k = 1,2
$$
$\mu$ 为总平均, $\alpha_i$为浓度为第$i$类的效应，$\beta_j$为温度为第$j$类的效应，$e_{ijk}$为随机误差

## (2) 在在显著性水平α=0.05下，检验浓度对产品得率的影响有无显著差异？

In [1]:
data <- data.frame(
  "观测值" = c(14, 10, 11, 11, 13, 19, 10, 12, 9, 7, 10, 8, 7, 11, 6, 10, 5, 11, 13, 14, 12, 13, 14, 10),
  "浓度" = factor(rep(c(2, 4, 6), each = 8)),
  "温度" = factor(rep(c(10,10,24,24,38,38,52,52),times = 3))
)
data

观测值,浓度,温度
<dbl>,<fct>,<fct>
14,2,10
10,2,10
11,2,24
11,2,24
13,2,38
19,2,38
10,2,52
12,2,52
9,4,10


In [2]:
anova_result <- aov(`观测值`~`浓度`+`温度`,data = data)
summary(anova_result)

            Df Sum Sq Mean Sq F value Pr(>F)  
浓度         2  69.33   34.67   5.490 0.0138 *
温度         3  32.33   10.78   1.707 0.2013  
Residuals   18 113.67    6.31                 
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

方差分析表为：

| 方差来源| 平方和   | 自由度  | 均方    | F比      |
|---------|---------|---------|---------|---------|
| 浓度    | 69.33   | 2       | 34.67   | 5.49    |
| 温度    | 32.33   | 3       | 10.78   |  1.71  |
| 误差    | 113.67  | 18      | 6.31    |          |
| 总和    | 215.33  | 23      |         |          |

浓度的p值0.0138小于0.05，所以在显著性水平α=0.05下，检验浓度对产品得率的影响有显著差异

## (3) 在显著性水平α=0.05下，检验温度对产品得率的影响有无显著差异？

温度的p值0.2013大于0.05，所以在显著性水平α=0.05下，检验温度对产品得率的影响没有显著差异